beacon-chain/rpc/prysm/v1alpha1/validator/duties.go

```diff
@@ -28,6 +31,17 @@ func (vs *Server) GetDuties(ctx context.Context, req *ethpb.DutiesRequest) (*eth
 	return vs.duties(ctx, req)
 }
 
+// Deprecated: The gRPC API will remain the default and fully supported through v8 (expected in 2026) but will be eventually removed in favor of REST API.
+//
+// GetDutiesV2 returns the duties assigned to a list of validators specified
+// in the request object.
```

Comment:
```suggestion
// GetDutiesV2 returns the duties assigned to a list of validators specified
// in the request object.
//
// Deprecated: The gRPC API will remain the default and fully supported through v8 (expected in 2026) but will be eventually removed in favor of REST API.
```

A godoc comment should start with the function name